### PR TITLE
Implement <AnswerCard> and its descendant components

### PR DESCRIPTION
**Pull request by axross**
localhost_51149__path=_docs_answercard--example (1)

### CODE REVIEW

**Comment by axross** [comment]
Could you have a look when you have time?

I think it's easier if you see [storybook](https://deploy-preview-153--csof.netlify.com/?path=/story/answercard--example) and review code at the same time.

File: components/AnswerCard/AnswerCardContentLoader.tsx [hunk 1]
```diff
@@ -0,0 +1,170 @@
+import * as React from "react";
```
Comment (ryotogashi) [inline components/AnswerCard/AnswerCardContentLoader.tsx:1]: niceeee!

File: components/AnswerCard/AnswerCardContentLoader.tsx [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+    </Root>
+  );
+}
+
+const Root = styled(ContentLoader)`
+  --avatar-diameter: 48px;
+  --margin-right-of-avatar: 16px;
+  --body-height: 16px;
+  --body-line-height: calc(16px * 1.333);
+  --body-vertical-padding: calc(
```
Comment (ryotogashi) [inline components/AnswerCard/AnswerCardContentLoader.tsx:43]: @axross Are you using this CSS valuable?

Comment (ryotogashi) [inline components/AnswerCard/AnswerCardContentLoader.tsx:43]: Are you using this CSS valuable?

File: components/AnswerCard/AnswerCardContentLoader.tsx [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+export interface Props extends React.Attributes {
+  className?: string;
+  style?: React.CSSProperties;
+  children?: React.ReactNode;
+}
+
+/**
+ * A part of `<AnswerCard>` that shows loading placeholder as answers or comments
+ */
+export default function AnswerCardContentLoader({ children, ...props }: Props) {
```
[the remark this answers is not in the document]
Comment (axross) [reply components/AnswerCard/AnswerCardContentLoader.tsx:13]: Thanks, that's unused. I'mma delete it

File: components/AnswerCard/AnswerCardContentLoader.tsx [hunk 4]
```diff
@@ -0,0 +1,170 @@
+import * as React from "react";
+import styled from "styled-components";
+import ContentLoader from "react-content-loader";
```
[the remark this answers is not in the document]
Comment (axross) [reply components/AnswerCard/AnswerCardContentLoader.tsx]: Thanks!

File: components/AnswerCard/AnswerCardContentLoader.tsx (hunk 2 above)
Comment (axross) [reply components/AnswerCard/AnswerCardContentLoader.tsx]: Deleted. Thanks!

**Pull request merged**